docs/axes/cartesian/linear.md

```diff
@@ -12,6 +12,8 @@ The following options are provided by the linear scale. They are all located in
 | `min` | `Number` | | User defined minimum number for the scale, overrides minimum value from data. [more...](#axis-range-settings)
 | `max` | `Number` | | User defined maximum number for the scale, overrides maximum value from data. [more...](#axis-range-settings)
 | `maxTicksLimit` | `Number` | `11` | Maximum number of ticks and gridlines to show.
+| `evenLabelSpacing` | `Boolean` |  | To be used with `maxTicksLimit`. If true, forces ticks to be evenly spaced, which is different to the default behaviour of unevenly spacing and forcing the display of the final point. 
+| `labelSpacing` | `Number` |  | This setting determines the spacing between labels on the x-axis.
```

Comment:
What's the difference between `labelSpacing` and `stepSize`? Is it that `labelSpacing` is used exactly whereas a multiple of `stepSize` may be used? If so, it may be better to reuse `stepSize` and see if we can do something like `autoSkip: false` instead